src/complexity.jl

```diff
@@ -0,0 +1,36 @@
+export ComplexityMeasure
+export complexity
+export complexity_normalized
+
+"""
+    ComplexityMeasure
+
+Abstract type for (entropy-like) complexity measures.
+"""
+abstract type ComplexityMeasure end
+
+"""
+    complexity(c::ComplexityMeasure, x)
+
+Estimate the complexity measure `c` for input data `x`, where `c` can be any of the
+following measures:
+
+- [`ReverseDispersion`](@ref).
+
+"""
+function complexity end
+
+"""
+    complexity_normalized(c::ComplexityMeasure, x) → m ∈ [a, b]
+
+Estimate the normalized complexity measure `c` for input data `x`, where `c` can
+can be any of the following measures:
+
+- [`ReverseDispersion`](@ref).
+
+The potential range `[a, b]` of the output value depends on `c`. See the documentation
+strings for the individual measures to get the normalized ranges.
```

Comment:
We should avoid duplicate information here. We shouldn't list the available measures. Instead, cross reference `complexity` docstring.

I also am not sure how much of value is the entire last sentence, and I would remove it.

But, what is even the normalized complexity mesure? In entropy this is clearly defined: the value of the entropy divided by the maximum possible value for the given estimator. Isn't it the same here? If so, maybe just say this?

Reply:
See my latest commit for a suggestion on an improve docstrings.